egs2/easycom/avsr1/README.md

```diff
@@ -0,0 +1,75 @@
+  If the face is not visible for some frames, we black out the frames so that we can still access the speech.
+  - feature extraction: By using frontends of pre-trained AV-HuBERT (visual frontend and audio frontend), we dump the audio-visual features.
+  - model training: Train the AVSR model, by initializing the encoder with AV-HuBERT.
+
+We can choose the following things.
+  - in `run.sh`
+    - av_hubert_model: Whether to use `base` model or `large` model of AV-HuBERT (Default: large)
+    - include_wearer: Whether include the data of eyeglasses wearer during test. (Please note the eyeglasses wearer in Easycom dataset has no face and only speech is available.) (Default: false)
+    - with_LRS3: Whether train the model with LRS3 dataset. As the Easycom dataset is small size, we can train with other avsr datasets to achieve proper performances. (Default: true)
+    - noise_augmentation: Whether applying noise augmentation (babble noise of NOISEX) during training. As we train the model with dumped features, it's not available to apply the noise in raw waveform. Instead, we apply the noise perturbation at the feature extracted by the audio frontend. (Default: true)
+
+  - in model configuration (`conf/*.yaml`)
+    - modality_dropout: If the value is greater than 0, then one of modalities (audio or video) is dropped out with the probability of the value.
+    - noise_augmentation: If it is set in `run.sh`, it should be set in the configuration.
+    - audio_only: Whether performing Audio-only speech recognition (ASR).
+
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Mon Jan 22 14:57:53 KST 2024`
+- python version: `3.9.18 (main, Sep 11 2023, 13:41:44)  [GCC 11.2.0]`
+- espnet version: `espnet 202310`
+- pytorch version: `pytorch 1.12.1`
+- Git hash: `a4c5547b1b484141e79cee6d972c41247c1cdaa3`
+  - Commit date: `Fri Dec 8 20:59:29 2023 -0500`
+- Pretrained model: [https://huggingface.co/espnet/msk_easycom_avsr_avhubert_large_extracted_en_bpe1000](https://huggingface.co/espnet/msk_easycom_avsr_avhubert_large_extracted_en_bpe1000)
+
+## Audio-Visual Speech Recognition Results (Multimodal) <br> exp/asr_train_avsr_avhubert_large_with_lrs3_noise_extracted_en_bpe1000
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|inference_asr_model_valid.acc.ave/test_with_LRS3|694|5572|70.5|22.4|7.1|5.5|35.0|75.4|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|inference_asr_model_valid.acc.ave/test_with_LRS3|694|26995|82.2|8.6|9.2|5.7|23.5|75.4|
+
+### TER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|inference_asr_model_valid.acc.ave/test_with_LRS3|694|8886|70.4|18.6|11.0|5.0|34.6|75.4|
+
+## Audio-only Speech Recognition Results (Audio-only) <br> exp/asr_train_avsr_avhubert_large_with_lrs3_noise_extracted_en_bpe1000
```

Comment:
The audio-only data significantly degrades the performance.
Can you provide some reasons?
Probably due to the AV HuBERT architecture?